cmd/rqlite/main.go

```diff
@@ -31,6 +31,7 @@ const cliHelp = `.help				Show this message
 .status				Show status and diagnostic information for connected node
 .expvar				Show expvar (Go runtime) information for connected node
 .tables				List names of tables
+.timer on|off			Turn SQL timer on or off
```

Comment:
The help message looks like it needs to be pushed over move, so it lines up with the messages above.

Reply:
I thought the same, but if I add any more tabs it won't be properly aligned. If you try to edit the file [here](https://github.com/joaodrp/rqlite/edit/add-timings-support-to-cli/cmd/rqlite/main.go?pr=/rqlite/rqlite/pull/414) it's aligned. Also, if you take a look at the sample session below, you'll see that all lines are aligned when using the CLI.